### PR TITLE
Draft: Add overdeliveries to logistics

### PR DESCRIPTION
**Pull request by ibzm**

### CODE REVIEW

**Comment by JacoMH** [comment]
SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`stock_hive`.`order_item`, CONSTRAINT `order_item_order_id_foreign` FOREIGN KEY (`order_id`) REFERENCES `order` (`id`) ON DELETE CASCADE) (Connection: mysql, SQL: insert into `order_item` (`item_id`, `order_id`, `ordered`, `price`) values (1, 1, 10, 5.99), (2, 1, 20, 3.49), (3, 2, 5, 12.99), (1, 3, 15, 4.99))

  at vendor\laravel\framework\src\Illuminate\Database\Connection.php:825
    821▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    822▕                 );
    823▕             }
    824▕
  ➜ 825▕             throw new QueryException(     
    826▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    827▕             );
    828▕         }
    829▕     }

eign` FOREIGN KEY (`order_id`) REFERENCES `order` (`id`) ON DELETE CASCADE)")

  2   vendor\laravel\framework\src\Illuminate\Database\MySqlConnection.php:53
      PDOStatement::execute()

this error was thrown when attempting to migrate your database

**Review by JacoMH — CHANGES_REQUESTED** [review]
SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`stock_hive`.`order_item`, CONSTRAINT `order_item_order_id_foreign` FOREIGN KEY (`order_id`) REFERENCES `order` (`id`) ON DELETE CASCADE) (Connection: mysql, SQL: insert into `order_item` (`item_id`, `order_id`, `ordered`, `price`) values (1, 1, 10, 5.99), (2, 1, 20, 3.49), (3, 2, 5, 12.99), (1, 3, 15, 4.99))

  at vendor\laravel\framework\src\Illuminate\Database\Connection.php:825
    821▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    822▕                 );
    823▕             }
    824▕
  ➜ 825▕             throw new QueryException(     
    826▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    827▕             );
    828▕         }
    829▕     }

eign` FOREIGN KEY (`order_id`) REFERENCES `order` (`id`) ON DELETE CASCADE)")

  2   vendor\laravel\framework\src\Illuminate\Database\MySqlConnection.php:53
      PDOStatement::execute() 

review your database, this error was thrown when attempting to build it on my machine

File: resources/views/Logistics/index.blade.php [hunk 1]
```diff
@@ -24,26 +21,27 @@
...
-             </td>
-            </tr>
-             @endforeach
-            </tbody>
+                            <td class="py-2 px-4">{{ $order->id }}</td>
+                            <td class="py-2 px-4">{{ $order->user_id }}</td>
+                            <td class="py-2 px-4">{{ $order->store_id }}</td>
+                            <td class="py-2 px-4">{{ $order->date_time }}</td>
+                            <td class="py-2 px-4 flex justify-center items-center gap-4">
+                                <x-primary-button>View Order</x-primary-button>
```
Comment (WebExtension1) [inline resources/views/Logistics/index.blade.php:29]: Make this link to Jacobs pre-existing View Order page

File: resources/views/Logistics/show.blade.php [hunk 2]
```diff
@@ -5,25 +5,29 @@
         </h2>
     </x-slot>
 
-  
     <div class="bg-stockhive-grey-dark text-white shadow-sm rounded-lg mt-8 lg:w-[85%] w-full m-auto p-6">
         <h1 class="text-2xl font-bold mb-4">Order ID: {{ $order->id }}</h1>
         <p><strong>User ID:</strong> {{ $order->user_id }}</p>
         <p><strong>Store ID:</strong> {{ $order->store_id }}</p>
```
Comment (WebExtension1) [inline resources/views/Logistics/show.blade.php:11]: Remove store id

File: resources/views/Logistics/show.blade.php [hunk 3]
```diff
@@ -5,25 +5,29 @@
         </h2>
     </x-slot>
 
-  
     <div class="bg-stockhive-grey-dark text-white shadow-sm rounded-lg mt-8 lg:w-[85%] w-full m-auto p-6">
         <h1 class="text-2xl font-bold mb-4">Order ID: {{ $order->id }}</h1>
         <p><strong>User ID:</strong> {{ $order->user_id }}</p>
```
Comment (WebExtension1) [inline resources/views/Logistics/show.blade.php:10]: Username instead of user id

**Review by WebExtension1 — CHANGES_REQUESTED** [review]
Other than these small changes, all looks good.

**Pull request merged**